### PR TITLE
Init of post api for grid_calendar

### PR DESCRIPTION
**Pull request by pbougue**
add a first POST api:

/grid_calendar
a file needs to be posted

TODO:

- [x] tests

### CODE REVIEW

File: tartare/api/grid_calendar.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+from flask.globals import request
+from flask_restful import Resource, Api
+from tartare import app
+
+api = Api(app)
+
+class GridCalendar(Resource):
+    def post(self):
+        content = request.data
+        print("content received: {}".format(content))
```
Comment (pbougue) [inline tartare/api/grid_calendar.py]: should use logger when PR #4 is merged

File: tartare/api.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
```
Comment (kinnou02) [inline tartare/api.py]: do you need it?

File: tartare/api/grid_calendar.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        return '', 200
+
+api.add_resource(GridCalendar, '/grid_calendar')
+
+app.run()
```
Comment (kinnou02) [inline tartare/api/grid_calendar.py]: :-1:

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,2 @@
+*.pyc
+*_pb2.py
```
Comment (antoine-de) [inline .gitignore]: why do we need protobuf ?

File: tartare/api/grid_calendar.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+from flask_restful import Resource, Api
+from tartare import app
+
+api = Api(app)
+
+class GridCalendar(Resource):
+    def post(self):
+        content = request.data
+        print("content received: {}".format(content))
+        return '', 200
```
Comment (antoine-de) [inline tartare/api/grid_calendar.py]: we can return a small json with maybe a `status`: OK

File: tartare/api/grid_calendar.py [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+api.add_resource(GridCalendar, '/grid_calendar')
```
Comment (antoine-de) [inline tartare/api/grid_calendar.py]: I think the routing should be in a separate file

**Comment by pbougue** [comment]
usage : PYTHONPATH=path/to/tartare python tartare/api.py

**Comment by antoine-de** [comment]
can't we use a `manage.py runserver` like tyr ?

**Comment by pbougue** [comment]
@antoine-de : ok, we will have a look at that, it would be better

File: .gitignore (hunk 4 above)
Comment (pbougue) [inline .gitignore]: to remove

File: tartare/api.py (hunk 2 above)
Comment (pbougue) [inline tartare/api.py]: nope

File: tartare/api/grid_calendar.py [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+# Stay tuned using
+# twitter @navitia
+# IRC #navitia on freenode
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+from flask.globals import request
+from flask_restful import Resource, Api
+from tartare import app
+
+api = Api(app)
```
Comment (kinnou02) [inline tartare/api/grid_calendar.py]: This should probably be created only one time

Comment (antoine-de) [inline tartare/api/grid_calendar.py]: yes we talked about moving this (and the routing) to api.py

Comment (pbougue) [inline tartare/api/grid_calendar.py]: moved in manage.py (I don't know if it's the best)
So now `./manage.py runserver` works, and `honcho start web` also

**Comment by buildbotctp** [comment]
Can one of the admins verify this patch?

**Comment by antoine-de** [comment]
Jenkins, test this please

**Comment by buildbotctp** [comment]
Build finished.

**Comment by antoine-de** [comment]
Jenkins, test this please

**Comment by antoine-de** [comment]
Jenkins, test this please

File: manage.py [hunk 8]
```diff
@@ -30,4 +30,14 @@
...
+from tartare.api.grid_calendar import GridCalendar
+import sys
+from flask_restful import Api
+
+api = Api(app)
+api.add_resource(GridCalendar, '/grid_calendar')
+
+if __name__ == '__main__':
+    app.config['DEBUG'] = True
+    app.run()
```
Comment (antoine-de) [inline manage.py]: I don't think it's the way it should be, we need a flask_script manager no ?

else the runserver of the cli is useless :wink:

File: tartare/api/grid_calendar.py (hunk 1 above)
Comment (antoine-de) [inline tartare/api/grid_calendar.py]: you can now do this

**Comment by datanel** [comment]
usage : 
pip install -r requirements_dev.txt
PYTHONPATH=path/to/tartare python tartare/api.py

**Comment by datanel** [comment]
I think you need **Flask-Script** in requirements.txt

**Comment by datanel** [comment]
you may add documentation "How to run the api"

File: manage.py [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+#
+# Stay tuned using
+# twitter @navitia
+# IRC #navitia on freenode
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+
+from tartare import app
+import sys
+from flask_script import Manager
```
Comment (kinnou02) [inline manage.py]: with flask 0.11 a command system is directly included in flask: http://flask.pocoo.org/docs/0.11/cli/#basic-usage

Comment (datanel) [inline manage.py]: PYTHONPATH=. python ./manage.py runserver
Traceback (most recent call last):
  File "./manage.py", line 35, in <module>
    from flask_script import Manager
ImportError: No module named 'flask_script'

I have Flask==0.11.1

Comment (pbougue) [inline manage.py]: Ok, so no manage.py needed, I remove it, and just update Profile

**Pull request merged**